smart-contract/assembly/__tests__/deweb-interface.spec.ts

```diff
@@ -107,14 +121,133 @@ describe('website deployer internals functions tests', () => {
     });
 
     throws('should throw if file does not exist', () => {
-      getChunk(chunkGetArgs(file1NameHash, 0));
+      getChunk(chunkGetArgs(file1PathHash, 0));
     });
   });
-});
 
-function chunkGetArgs(
-  filePathHash: StaticArray<u8>,
-  index: u32,
-): StaticArray<u8> {
-  return new ChunkGet(filePathHash, index).serialize();
-}
+  // Testing delete files
+  describe('Delete File', () => {
+    beforeEach(() => {
+      resetStorage();
+      setDeployContext(user);
+      constructor(new Args().serialize());
+    });
+
+    test('Should delete 1 file with 1 chunk', () => {
+      const myUpload = given()
+        .withFile(file1Path, 1, [fileData1])
+        .preStore()
+        .storeAll();
+
+      const fileToDelete = new ChunkDelete(
+        file1Path,
+        sha256(stringToBytes(file1Path)),
+      );
+
+      myUpload.deleteFile(fileToDelete);
+
+      throws('should throw if file does not exist', () => {
+        getChunk(chunkGetArgs(file1PathHash, 0));
+      });
```

Comment:
Could add a method `checkThat(myUpload).throwsIfNoChunk()`

Reply:
i tried but i encountered some un-expected behavior, I will try again

Reply:
AS doesn't like it, we can discuss the details @Ben-Rey